Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>used as a prior for standard deviations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>This ones already mentioned twice so omitting the text as I'll speak to it</p>